pantahub.c

```diff
@@ -152,7 +152,7 @@ const char** pv_ph_get_certs(struct pantavisor *__unused)
 		return NULL;
 
 	// Always n-1 due to . and .., and need one extra
-	cafiles = calloc(1, (sizeof(char*) * (n-1)));
+	cafiles = calloc(n - 1, sizeof(char*));
```

Comment:
I guess just char here to keep consistence

Reply:
cafiles is an array of char* no? looks correct to me ...

Reply:
yes, cafiles is a array of char*

Reply:
oh, sorry. nevermind